src/Main.hs

```diff
@@ -45,7 +44,7 @@ main = do
             Rules.buildRules
             Rules.Documentation.documentationRules
             Rules.Clean.cleanRules
-            Rules.Install.installRules
+            -- Rules.Install.installRules
```

Comment:
What's up with `installRules`? If they are broken, we need to open an issue with an explanation what needs to be done and a comment pointing to this issue.